signinwithapplebutton/src/main/java/com/willowtreeapps/signinwithapplebutton/view/SignInWithAppleButton.kt

```diff
@@ -60,14 +59,14 @@ class SignInWithAppleButton @JvmOverloads constructor(
         (background as? GradientDrawable)?.cornerRadius = cornerRadius
 
         if (icon != -1) {
-            imageView.setImageResource(icon)
+            textView.setCompoundDrawablesRelativeWithIntrinsicBounds(icon, 0, 0, 0)
 
             if (iconSize != -1) {
-                imageView.layoutParams.width = iconSize
-                imageView.layoutParams.height = iconSize
+                textView.layoutParams.width = iconSize
+                textView.layoutParams.height = iconSize
             }
         } else {
-            imageView.visibility = View.GONE
+            textView.visibility = View.GONE
```

Comment:
Ditto here, this visibility change was to hide the icon if its size was not set. We're always showing the TextView, so we don't need to consider a case where it gets hidden.